include/db/object.h

```diff
@@ -60,6 +63,8 @@ class LocalObject : public QEnableSharedFromThis<LocalObject> {
   virtual dbif::ObjectType type() const = 0;
   QString name() const { return name_; }
   QString comment() const { return comment_; }
+  uint64_t id() const { return id_; }
+  const QSet<PLocalObject> &children() { return children_; }
```

Comment:
nitpick: in other places you do const X& x(), not const X &x() (I like the first one more)